tests/lsicorr.py

```diff
@@ -0,0 +1,164 @@
+import os
+import unittest
+import time
+
+from parameterized import parameterized
+
+from utils.channel_access import ChannelAccess
+from utils.ioc_launcher import IOCRegister, get_default_ioc_dir, EPICS_TOP, PythonIOCLauncher
+from utils.test_modes import TestModes
+from utils.testing import parameterized_list, ManagerMode, unstable_test
+
+ioc_number = 1
+DEVICE_PREFIX = "LSICORR_{:02d}".format(ioc_number)
+
+LSICORR_PATH = os.path.join(EPICS_TOP, "support", "lsicorr", "master")
+IOCS = [
+    {
+        "ioc_launcher_class": PythonIOCLauncher,
+        "name": DEVICE_PREFIX,
+        "directory": LSICORR_PATH,
+        "python_script_commandline": [os.path.join(LSICORR_PATH, "LSi_Correlator.py"), "--pv_prefix", "TE:NDW1836:", "--ioc_number", "{}".format(ioc_number)],
+        "started_text": "IOC started",
+        "python_version": 3,
```

Comment:
Because the IOC has no iocsh the normal IOC launcher can't kill the IOC after the tests have ran. I've changed it to work with the procserv launcher instead, which still boots it in a more 'normal' way